plugins/UsersManager/Controller.php

```diff
@@ -458,6 +459,31 @@ public function recordUserSettings()
         return $toReturn;
     }
 
+    /**
+     * Regenerates a users API authentication token.
+     * @throws Exception
+     */
+    public function regenerateTokenAuth()
```

Comment:
Is it instead possible to directly call `UsersManager.regenerateTokenAuth` API from JavaSCript?